distatis_weights_upgma.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Run this file using 'qsub distatis_weights_upgma.sh path/to/input.fasta path/to/output_alignment.txt path/to/output_matrix_test.txt path/to/output_matrix.csv'
+
+# All lines starting with "#$" are SGE qsub commands
+
+# Specify a project to run under
+#$ -P bi720
+
+# Give this job a name
+#$ -N distatis_weights_upgma
+
+# Join standard output and error to a single file
+#$ -j y
+
+# Name the file where to redirect standard output and error
+#$ -o /projectnb/bi720/MMG/2fungi2furious/hpenn13/compbio-project/logs/distatis_weights_upgma.log
+
+# Send an email when the job finishes
+#$ -m eas
+
+# Request a large memory node, this will affect your queue time, but it's better to overestimate
+# -l mem_per_core=8G
+
+# Request more cores, this will affect your queue time, make sure your program supports multithreading, or it's a waste
+# -pe omp 1
+
+# Now we write the script that the compute node will work on.
+
+source ./common.sh
+prerun
+
+# load modules
+module load R/4.4.0
+
+# do some work
+datafolder="/projectnb/bi720/MMG/compbio"
```

Comment:
I _think_ this is included in the `common.sh` file to avoid having to redefine in every qsub